### ISSUE TITLE
Error validating signature when installing extensions from OpenVSX

### ISSUE THREAD

**Issue by jmcphers** [posit-dev/positron#5758]
#### Positron and OS details:

```
Positron Version: 2025.01.0 build 87
Code - OSS Version: 1.95.0
Commit: 240e51fa165f6c512e586f0af105a0c8fc092607
Date: 2024-12-16T17:23:23.892Z
Electron: 32.2.1
Chromium: 128.0.6613.186
Node.js: 20.18.0
V8: 12.8.374.38-electron.0
OS: Darwin arm64 24.1.0
```

#### Interpreter details:

N/A

## Describe the issue:

When attempting to install an extension from OpenVSX, a signature validation error occurs.

## Steps to reproduce the issue:

1. Go to the Extensions panel
2. Search for an extension, e.g. Posit Publisher
3. Click the Install button

<img width="1519" alt="Image" src="https://github.com/user-attachments/assets/d19323d3-be06-4f21-895c-d6fd03ab8ad6" />

## Expected or desired behavior:

Extension installs successfully.

## Were there any error messages in the UI, Output panel, or Developer Tools console?

This looks related:

```
2024-12-16 10:12:38.400 [error] SignatureVerificationInternal: Signature verification was not executed.
    at ExtensionManagementService2.downloadExtension (file:///Users/jmcphers/git/VSCode-darwin-arm64/Positron.app/Contents/Resources/app/out/vs/code/electron-utility/sharedProcess/sharedProcessMain.js:25465:15)
    at async ExtensionManagementService2.downloadAndExtractGalleryExtension (file:///Users/jmcphers/git/VSCode-darwin-arm64/Positron.app/Contents/Resources/app/out/vs/code/electron-utility/sharedProcess/sharedProcessMain.js:25412:46)
2024-12-16 10:12:38.694 [error] Signature verification was not executed.: SignatureVerificationInternal: Signature verification was not executed.
    at ExtensionManagementService2.downloadExtension (file:///Users/jmcphers/git/VSCode-darwin-arm64/Positron.app/Contents/Resources/app/out/vs/code/electron-utility/sharedProcess/sharedProcessMain.js:25465:15)
    at async ExtensionManagementService2.downloadAndExtractGalleryExtension (file:///Users/jmcphers/git/VSCode-darwin-arm64/Positron.app/Contents/Resources/app/out/vs/code/electron-utility/sharedProcess/sharedProcessMain.js:25412:46)
```

...

```
2024-12-16 10:17:14.168 [error] Cannot find package 'yauzl' imported from /Users/jmcphers/git/VSCode-darwin-arm64/Positron.app/Contents/Resources/app/out/vs/code/electron-utility/sharedProcess/sharedProcessMain.js: Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'yauzl' imported from /Users/jmcphers/git/VSCode-darwin-arm64/Positron.app/Contents/Resources/app/out/vs/code/electron-utility/sharedProcess/sharedProcessMain.js
    at packageResolve (node:internal/modules/esm/resolve:861:9)
    at moduleResolve (node:internal/modules/esm/resolve:934:18)
    at defaultResolve (node:internal/modules/esm/resolve:1176:11)
    at nextResolve (node:internal/modules/esm/hooks:868:28)
    at resolve (data:text/javascript;base64,CglleHBvcnQgYXN5bmMgZnVuY3Rpb24gcmVzb2x2ZShzcGVjaWZpZXIsIGNvbnRleHQsIG5leHRSZXNvbHZlKSB7CgkJaWYgKHNwZWNpZmllciA9PT0gJ2ZzJykgewoJCQlyZXR1cm4gewoJCQkJZm9ybWF0OiAnYnVpbHRpbicsCgkJCQlzaG9ydENpcmN1aXQ6IHRydWUsCgkJCQl1cmw6ICdub2RlOm9yaWdpbmFsLWZzJwoJCQl9OwoJCX0KCgkJLy8gRGVmZXIgdG8gdGhlIG5leHQgaG9vayBpbiB0aGUgY2hhaW4sIHdoaWNoIHdvdWxkIGJlIHRoZQoJCS8vIE5vZGUuanMgZGVmYXVsdCByZXNvbHZlIGlmIHRoaXMgaXMgdGhlIGxhc3QgdXNlci1zcGVjaWZpZWQgbG9hZGVyLgoJCXJldHVybiBuZXh0UmVzb2x2ZShzcGVjaWZpZXIsIGNvbnRleHQpOwoJfQ==:13:10)
    at nextResolve (node:internal/modules/esm/hooks:868:28)
    at Hooks.resolve (node:internal/modules/esm/hooks:306:30)
    at MessagePort.handleMessage (node:internal/modules/esm/worker:196:24)
    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:820:20)
    at MessagePort.<anonymous> (node:internal/per_context/messageport:23:28)
```

**Comment by jmcphers** [comment]
I think this may have to do with this commit from upstream. This landed in 1.94 (which was included in our upgrade from 1.93 -> 1.95) and blocked installation of VSIX files that could not validated.

https://github.com/microsoft/vscode/commit/2991008c4fd8ba016ed5d25765f816f126cba525

Later, in 1.96, this was walked back a bit to not validate unsigned VSIX files. 

https://github.com/microsoft/vscode/commit/6b2777169df77698e39a38033de20b73e9e95561

However as we're on 1.95 we don't have this change.

**Comment by testlabauto** [comment]
#### Verified Fixed
    Positron Version(s) : 2025.01.0-104
    OS Version          : OSX

### Test scenario(s) 

Posit Publisher installed successfully

Link(s) to TestRail test cases run or created:
